pkg/util/util.go

```diff
@@ -35,3 +35,15 @@ func OutputErr(err error) error {
 	}
 	return err
 }
+
+// ShortenString shortens a string to the specified length, replacing
+// newlines with spaces. If the string is longer than length, it appends
+// a unicode ellipsis character. If length is 0, it returns the unmodified
+// string.
+func ShortenString(msg string, length int) string {
```

Comment:
(nitpick) I'm not sure we should move it out, the side-effect of \n->" " for a general util is a bit weird.

If we move it out, maybe into `strutil` to not have a kitchen-sink `util`? I think you mention that you generally preferred it this way in another PR.  And maybe `strutil.ShortenAndNormalize()` or something (this one is a bit long) to indicate that there is this other side-effect

Reply:
Its fine as is, thanks for the tweaks about the `\n`